docker/db/init

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# customize postgres startup here
+# 
+# Synopsis: /init secret [username]
+#  or starting current state: /init
```

Comment:
Based on the init script in the FROM image.  That one only took the 'secret' for the 'root' user.  We add the ability to specify the user name as a second argument.